app/assets/javascripts/tagging.js

```diff
@@ -1,10 +1,19 @@
 function addTag(tagname, selector) {
 
   selector = selector || '#tagform';
+  if (tagname.indexOf("place") != -1) {
+    place = tagname.split(":")[1];
+    place.replace("-", " ");
+
+    confirm("This looks like a location. Is this full description of the location accurate?");
+    if(confirm) {
```

Comment:
aha - so here, `confirm()` returns  true or false, but doesn't create a `confirm` variable. You can just say:

```js
if (confirm("......")) {
```

Or you could say:

```js
var confirm = confirm("......");
if (confirm) {
```

Make sense?

Reply:
Yes, done!